std-bits/table/src/main/java/org/enso/table/data/table/Column.java

```diff
@@ -161,38 +161,31 @@ public static Column fromItemsNoDateConversion(String name, List<Object> items,
   }
 
   /**
-   * Creates a new column with given name and elements.
+   * Creates a new column with given name and an element to repeat.
    *
    * @param name the name to use
-   * @param items the items contained in the column
+   * @param items the item repeated in the column
    * @return a column with given name and items
    */
-  public static Column fromRepeatedItems(String name, List<Value> items, int repeat, ProblemAggregator problemAggregator) {
-    if (repeat < 1) {
-      throw new IllegalArgumentException("Repeat count must be positive.");
+  public static Column fromRepeatedItem(String name, Value item, int repeat, ProblemAggregator problemAggregator) {
+    if (repeat < 0) {
+      throw new IllegalArgumentException("Repeat count must be non-negative.");
+    } else if (repeat == 1) {
+      return fromItems(name, List.of(item), null, problemAggregator);
     }
 
-    if (repeat == 1) {
-      return fromItems(name, items, null, problemAggregator);
-    }
+    var builder = new InferredBuilder(repeat, problemAggregator);
+    Object converted = item instanceof Value v ? Polyglot_Utils.convertPolyglotValue(v) : item;
```

Comment:
I thought we agreed in the discussion that we don't want to use the `InferredBuilder` for this, but instead use `StorageType::forBoxedItem + Builder::getForType`.